### PR TITLE
ER-1494: Fixed handling of HTML entities in content

### PR DESCRIPTION
**Pull request by rimi-itk**
#### Link to ticket

https://jira.itkdev.dk/browse/ER-1494

#### Description

Fixes handling of quotes (`&quot`) and other exotic characters in feed content.

#### Checklist

- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change
should be exempt from the list.

### CODE REVIEW

File: sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php [hunk 1]
```diff
@@ -852,7 +852,7 @@ public function getLinkBoxes(\ParagraphsItemEntity $paragraph) {
                   'type' => 'link_box',
                   'button' => $link['title'],
```
Comment (cableman) [inline sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php:853]: How about button title?

File: sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php [hunk 2]
```diff
@@ -820,7 +820,7 @@ public function getLinkBoxes(\ParagraphsItemEntity $paragraph) {
...
```
Comment (cableman) [inline sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php:821]: How about button title?

File: sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php [hunk 3]
```diff
@@ -531,7 +531,7 @@ private function getLink(\ParagraphsItemEntity $paragraph) {
     return [
       'guid' => $this->getGuid($paragraph),
       'type' => $this->getType($paragraph),
-      'title' => $this->getTitle($this->nodeHelper->getFieldValue($paragraph, 'field_link', 'title')),
+      'title' => $this->getContent($this->nodeHelper->getFieldValue($paragraph, 'field_link', 'title')),
       'url' => $this->getAbsoluteUrl($this->nodeHelper->getFieldValue($paragraph, 'field_link', 'url')),
       'color' => ltrim($this->nodeHelper->getFieldValue($paragraph, 'field_link_color', 'rgb'), '#'),
       'subtitle' => $this->nodeHelper->getFieldValue($paragraph, 'field_link_text', 'value'),
```
Comment (cableman) [inline sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php:537]: How about subtitle?

**Review by cableman — APPROVED** [review]
Approved with minor question

File: sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php (hunk 1 above)
Comment (rimi-itk) [reply sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php]: Good point.

**Review by cableman — APPROVED** [review]

**Pull request merged**